easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.09-foss-2019b.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2019.09'
+
+homepage = 'https://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+multi_deps = {'Python': ['3.7.4', '2.7.16']}
+
+use_pip = True
+
+# order is important!
+# package versions updated Sep 17th 2019 to latest versions compatible with python2
```

Comment:
so, this is the big question, for how long do we want to hold back python3 for the sake of python2?

Reply:
I'm kind of torn between having a separate `SciPy-bundle` for Python 2 (with the extension versions used here) and one for Python 3 (using the latest versions), with a `versionsuffix`.

Using different extension versions depending on the Python version in a single `SciPy-bundle` easyconfig using `multi_deps` could be confusing. Also, it's currently not possible in a clean way, we'd have to make some enhancements in framework for that (and I'm not sure that i) I'm a big fan of that, ii) how that should look like).

So, my vote goes to having a separate `SciPy-bundle` easyconfig for Python 2 & 3.

@easybuilders/easybuild-easyconfigs-maintainers What do the other maintainers think?

Reply:
@boegel will we be able to have multi_deps that has dependencies that use versionsuffixes? Because there probably still are quite a few of those cases, which we would then have to split as well (but i guess it's inevitable, more and more of them are probably going to also drop python2 in the near future).

Reply:
@Micket Adding support for `multi_deps` to specify `SciPy-bundle` with a `versionsuffix` included shouldn't be too difficult.

Reply:
I prepared separate easyconfigs for Python2 & 3 using versionsuffix. The Python 3 version contains up-to date components. I have successfuly compiled VTK and ParaView with this new Python 3 version.